### PR TITLE
[NewAPI] Fix multiple transfers + server lie on pokestop distance

### PR DESCRIPTION
**Pull request by FrancYescO**
- [x] Fix This:

- [ ] And This (nope.. https://github.com/jabbink/PokemonGoBot/pull/1242#issuecomment-242915452):

### CODE REVIEW

**Comment by FrancYescO** [comment]
About the pokestop thing .... seems like a softban or api mess:

File: src/main/kotlin/ink/abb/pogo/scraper/tasks/LootOneNearbyPokestop.kt [hunk 1]
```diff
@@ -73,7 +74,11 @@ class LootOneNearbyPokestop(val sortedPokestops: List<Pokestop>, val lootTimeout
                     lootTimeouts.put(closest.id, closest.cooldownCompleteTimestampMs)
                 }
                 Result.OUT_OF_RANGE -> {
-                    Log.red("Pokestop out of range; distance: ${closest.distance}")
+                    Log.red("Pokestop out of range; our calculated distance: ${DecimalFormat("#0.00").format(closest.distance)}m")
+                    if (closest.distance < 40) {
```
Comment (jabbink) [inline src/main/kotlin/ink/abb/pogo/scraper/tasks/LootOneNearbyPokestop.kt]: api.mapSettings has this distance in a non hardcoded way

Comment (FrancYescO) [inline src/main/kotlin/ink/abb/pogo/scraper/tasks/LootOneNearbyPokestop.kt]: .getMapObjectsMinDistanceMeters returns 10
.encounterRangeMeters returns 50

both seems wrong

Comment (jabbink) [inline src/main/kotlin/ink/abb/pogo/scraper/tasks/LootOneNearbyPokestop.kt]: Sorry. Fortsettings.interactiondistance

The one Pokestop.inRange() uses

Comment (FrancYescO) [inline src/main/kotlin/ink/abb/pogo/scraper/tasks/LootOneNearbyPokestop.kt]: ok got it and pushed

**Comment by FrancYescO** [comment]
The pokestop out of range seems happening after ~15min, having 2 account in the same instance (and both still alive after 15min started to have the pokestop out of range issue one just with few seconds after the other)

**Comment by FrancYescO** [comment]
Merged in https://github.com/jabbink/PokemonGoBot/pull/1237/commits/2950c595e0ddfdc33e213674c7178ecf9591fd0b

**Pull request closed without merging**